### ISSUE TITLE
less source mapping Classes/Service/CompileService.php

### ISSUE THREAD

**Issue by eberhardloh** [benjaminkott/bootstrap_package#413]
source mapping seemed not to work correctly, I made the follwing change:

```
<before>
                    // enable source mapping
                    $optionsForSourceMap = array(
                        'sourceMap' => true,
                        'sourceMapWriteTo' => GeneralUtility::getFileAbsFileName('typo3temp/bootstrappackage') . '/bootstrappackage.map',
                        'sourceMapURL' => '/typo3temp/bootstrappackage/bootstrappackage.map',
                        'sourceMapBasepath' => PATH_site,
                        'sourceMapRootpath' => '/'
                    );
</before>

```
```
<after>
				// enable source mapping
				$mapFilename = basename($file) . '.map';
				$optionsForSourceMap = array(
					'sourceMap' => true,
					'sourceMapWriteTo' => GeneralUtility::getFileAbsFileName('typo3temp/bootstrappackage') . "/{$mapFilename}",
					'sourceMapURL' => "/typo3temp/bootstrappackage/{$mapFilename}",
					'sourceMapBasepath' => PATH_site,
					'sourceMapRootpath' => '/'
				);
</after>

```

**Comment by D1ll1ng** [comment]
This did't do it for me. but mine looks a bit different as well.

`                    $optionsForSourceMap = [
                        'sourceMap' => true,
                        'sourceMapWriteTo' => GeneralUtility::getFileAbsFileName($this->tempDirectory) . 'bootstrappackage.map',
                        'sourceMapURL' => $this->tempDirectory . 'bootstrappackage.map',
                        'sourceMapBasepath' => PATH_site,
                        'sourceMapRootpath' => '/'
                    ];
                    $arguments['options'] += $optionsForSourceMap;
                   `

**Comment by chequille** [comment]
As I found, sitemap is not working on firefox and chrome.
I changed in less.php file following line
		if( $sourceMapUrl ){
			$output->add( sprintf('/*# sourceMappingURL=/%s */', $sourceMapUrl) );
		}
adding the slash before %s
After this in the css fille the following entry appears:
/*# sourceMappingURL=/typo3temp/boostrappackage/bootstrappackage.map */

Result: firefox is still not showing the source files but chrome is showing them now.

Any idea why firefox is not doing the job?
